build/postgres-ha/Dockerfile

```diff
@@ -117,38 +75,33 @@ RUN useradd crunchyadm -g 0 -u 17
 
 ENV PATH="${PGROOT}/bin:${PATH}"
 
-RUN mkdir -p /opt/cpm/bin /opt/cpm/conf /pgdata /pgwal /pgconf /backrestrepo \
+RUN mkdir -p /opt/crunchy/bin /opt/crunchy/conf /pgdata /pgwal /pgconf /backrestrepo \
```

Comment:
If these directories exist in the `crunchy-postgres` image do they need to be created or updated (with `chown` or `chmod`) in this image

Reply:
They shouldn't need to be, but TIAS as they say :wink:

Reply:
From what i've seen they do not need to be created in the child image. Ill remove the duplicates from `postgres-ha`